DSharpPlus.CommandsNext/CommandsNextExtension.cs

```diff
@@ -293,15 +292,8 @@ public Command FindCommand(string commandString, out string rawArguments)
                 if (next == null)
                     break;
 
-                if (ignoreCase)
-                {
-                    next = next.ToLowerInvariant();
-                    cmd = cm2.Children.FirstOrDefault(x => x.Name.ToLowerInvariant() == next || x.Aliases?.Any(xx => xx.ToLowerInvariant() == next) == true);
-                }
-                else
-                {
-                    cmd = cm2.Children.FirstOrDefault(x => x.Name == next || x.Aliases?.Contains(next) == true);
-                }
+                var comparison = ignoreCase ? StringComparison.InvariantCultureIgnoreCase : StringComparison.InvariantCulture;
+                cmd = cm2.Children.FirstOrDefault(x => x.Name.Equals(next, comparison) || x.Aliases?.Any(xx => xx.Equals(next, comparison)) == true);
```

Comment:
For this one we need to execute command that is in command group. With different casing. 
`d# biNd MsG 939952915488075937`
`d# bind message 939952915488075937`
![image](https://user-images.githubusercontent.com/50947930/152696417-295cda92-b81f-4cc7-8140-965b6944ff6c.png)
When `CaseSensitive` is set to true in bot config, it will work fine with `d# bind message 939952915488075937` and with `d# bind MsG 939952915488075937` it will find group fine but will fail to find command as it's name has different casing. Not sure how to properly show this on screenshot.